style.css

```diff
@@ -0,0 +1,718 @@
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-clip: content-box;
+  -webkit-background-clip: text;
+}
+
+.introduction-slogan {
+  font-family: "lato", sans-serif;
+}
+
+.introduction-description {
+  width: 100%;
+  background-color: #e7e7e7;
+  border: #fff solid 2px;
+  color: #58595a;
+  padding: 0.8rem;
+  font-family: "lato", sans-serif;
+}
+
+.introduction-offers {
+  color: #272a31;
+  font-weight: 700;
+  font-size: 27px;
+}
+
+.introduction-time {
+  color: #58595a;
+  font-weight: 500;
+}
+
+.container-1 {
+  background-image: url("https://clauzzen.files.wordpress.com/2011/01/vintage-pattern_black.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  width: auto;
+}
+
+.program-item-holder {
+  margin: 10% 3% 0 3%;
+  list-style: none;
+  padding: 0;
+}
+
+.item {
+  display: flex;
+  align-items: center;
+  background-color: #444242;
+  justify-content: space-between;
+  gap: 2%;
+  color: #d3d3d3;
+  margin-bottom: 10px;
+  margin-left: 10px;
+  list-style: none;
+  padding: 5% 5%;
+  opacity: 0.8;
+  margin-right: 10px;
+  transition: outline 0.15s ease-in-out;
+}
+
+.program-description {
+  color: #fff;
+  font-family: "Lato", sans-serif;
+  font-size: 12px;
+  letter-spacing: 1px;
+  flex-shrink: 1;
+}
+
+.item:hover {
+  outline: 2px solid #d3d3d3;
+}
+
+.item-icon {
+  padding-right: 4%;
+  margin: 15px 0;
+}
+
+.main-program {
+  font-family: "Cocogoose", sans-serif;
+  color: #d3d3d3;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.program-title-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.program-subtitle {
+  font-size: 1rem;
+  font-family: "Cocogoose", sans-serif;
+  color: #ec5242;
+}
+
+.hr-title {
+  border: 1px solid #ec5242;
+  width: 20%;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+  margin: 40px 0 20px 0;
+}
+
+.program-button {
+  padding: 5% 7%;
+  font-family: "lato", sans-serif;
+  font-size: 16px;
+  font-weight: 600;
+  margin: 0 auto;
+  background-color: #ec5242;
+  color: #f0f2f5;
+  border: 0;
+}
+
+.see-more {
+  display: none;
+}
+
+.featured-specialists {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 2rem 0;
+  width: 100%;
+}
+
+.specialist-heading {
+  font-size: 24px;
+  color: #272a31;
+  font-family: "Cocogoose", sans-serif;
+}
+
+.featured-specialists-container {
+  display: flex;
+  flex-direction: column;
+  margin: 0 3%;
+}
+
+.specialist-content {
+  display: flex;
+  gap: 5%;
+  padding: 1.5rem;
+  background-image: url("./images/chess.svg");
+  background-repeat: no-repeat;
+  background-size: 70px;
+  background-position: left top;
+}
+
+.specialist-image {
+  position: relative;
+  background-repeat: no-repeat;
+  background-size: 55%;
+  margin-top: 30px;
+  width: 115px;
+  height: 115px;
+  margin-right: 2%;
+}
```

Comment:
- [ ] I think it will be better if you fix the image position with the background image to match the given design, currently what we have :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217488513-38a66048-399e-44eb-9f05-3497b15d88fb.png)

How it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217488567-eddbb0d2-c491-485a-8fe3-9f9fdb68320f.png)

The easy fix would be to remove the margin top and right that will make it better :+1: